### DISCUSSION TITLE
Is merging h3 and p supported

### DISCUSSION THREAD

**Discussion by syabro** [codex-team/editor.js#1929] [Q&A]
**Question**
I'm trying to merge P and H3 on https://editorjs.io demo. Is it possible to merge them?

https://user-images.githubusercontent.com/29029/114311082-5dbd7480-9afe-11eb-94aa-eda66685e42e.mov

**Comment by gohabereg** [comment]
It's not possible at the moment because merge strategy is provided by the Tool and in general Tool doesn't know about data format of another Tool. 
However I think we can convert data formats using conversion config now, what do you think @neSpecc @khaydarov ?
